src/modules/privacy-idea-administration/privacy-idea-administration.service.spec.ts

```diff
@@ -832,4 +837,93 @@ describe(`PrivacyIdeaAdministrationService`, () => {
             expect(result).toBe(requires2fa);
         });
     });
+
+    describe('deleteUser', () => {
```

Comment:
names of the describe-blocks are switched around (the first one is testing the event handler)